Makefile

```diff
@@ -514,7 +528,17 @@ endif
 	docker run $(NODE_CONTAINER_NAME) calico-felix --version
 	docker run $(NODE_CONTAINER_NAME) libnetwork-plugin -v
 
-	@echo "\nNow push the tag and images. Then create a release on Github and attach the dist/calicoctl binary"
+	@echo "\nNow push the tag and images. Then create a release on Github and attach dist/calicoctl, dist/calicoctl-darwin-amd64, and dist/calicoctl-windows-amd64.exe binaries"
+	@echo "\nAdd release notes for calicoctl and calico/node. Use this URL to find commit messages for this release: https://github.com/projectcalico/calicoctl/compare/<old_release_version>...$(VERSION)"
```

Comment:
Could do `git log --oneline <old_release_version>...$(VERSION)` locally or some variant of that?